### ISSUE TITLE
Add one or two custom fields along with a button to copy to clipboard

### ISSUE THREAD

**Issue by GoodGuy98** [keepassxreboot/keepassxc#3738]
I would like this feature because sometimes signing on requires entering additional information. Currently I use the comments field and have to manually select the data I want to paste to the clipboard.  It is mixed in with the actual comments so it doesn't stand out quickly.

**Comment by droidmonkey** [comment]
You want to do to the advanced tab when editing the entry and add attributes using the to part of the page.

**Comment by GoodGuy98** [comment]
Thank you for the quick reply. The attributes feature is much better than the request I made. It took me a few minutes to realize the ability to copy each field existed on the right-click menu.

I am used to using a product called Any Password Pro for Windows which I run it under CrossOver Linux so I can share it between several OSes. I would love to convert it to KeePassXC but I made mistakes when I started using it so it would be a lot of work to migrate now.

Thank you again. I will mark it closed.

**Comment by droidmonkey** [comment]
If you can export to csv you could probably carry over the majority of entries by importing csv in KeePassXC

**Comment by GoodGuy98** [comment]
Any Password Pro does have the capability to export to a csv file but it won't work for me.
I have been using that program for years and I made a major mistake in how I set up the structure which I will explain below.

I created a folder for each website with individual password entries below that. I will show I did it for several years before learning the error of my ways when I found KeePassXC. I'll use KeePassXC as an example:

KeePassXC GoodGuy98 (Folder Name)
    Main Site GoodGuy98 (Password Entry Name)
    Forums GoodGuy98 (Password Entry Name)

When Any Password Pro exports to a cvs file it doesn't import enough information to allow me to know what the entries apply to. It would need to export the folder info to create the folder in KeePassXC as well as the password entries which it does now,

Now I create the setup like this:

KeePassXC GoodGuy98 (Folder Name)
    KeePassXC Main Site GoodGuy98 (Password Entry Name)
    KeePassXC Forums GoodGuy98 (Password Entry Name)

The program says I have 633 entries and their software has another strange issue.
It won't allow me to press F2 - Press Ctrl-C and then go to the password entries and paste it in front with Ctrl-V. It only works with the mouse. I am disabled from sever spinal stenosis and in constant pain. It will increase the pain and I have a hard enough time coping with it now.

Hopefully I explained it enough to make sense.

The software wasn't updated for years (payware) but they finally offered a newer version for 50% off. The produced up to RC 3 and it's been that version for months which isn't encouraging. The new version still doesn't support Ctrl-C and Ctrl-V in entries.

Thank you for your help.
